README.md

```diff
@@ -5,3 +5,32 @@
 
 This repository contains the Strimzi UI and its implementation.
 Strimzi UI provides a way for managing Strimzi and Kafka clusters (+ other components) deployed by it using a graphical user interface.
+
+## Getting started
+
+This UI uses `npm` to provide dependency management. If you wish to develop the UI, you will need:
+
+- [npm version 6.4.1 or later](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm)
+- [node 10.15.0 or later](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm)
```

Comment:
Can we fix to either latest or LTS? And then we can reflect that into the docker image as needed later on.

Reply:
These are the levels I personaly built the above with (which I know work). More than happy to bump these - worth an issue to do in your opinion to discuss more widley/implement the result?